chapter.md

```diff
@@ -0,0 +1,781 @@
+# Creating Functional Forms
+
+There are many different definitions of the term "software architecture".
+Our favorite is that
+a system's architecture is what you draw on the whiteboard
+when you're explaining it to a new developer.
```

Comment:
The flow from the first to the second sentence is somewhat confusing, I had to read it a few times. Something about the "our favourite" dividing those two thoughts...